docs/user/YAL5-codon-pairs_usage.md

```diff
@@ -0,0 +1,53 @@
+# Running YAL5-codon-pairs.R
+
+## Input ##
+The script `YAL5-codon-pairs.R` looks into metafeatures, currently the translational landscape surrounding codon pairs. 
+
+- h5 file
+- gff3 file
+- tsv codon positions file
+
+Read counts are fetched from the H5 file of a sample and assigns the counts to the cooresponding codon pair position for the genes using the species-specific GFF3 file and a tsv file containing the codon positions for all genes of the species being studied, i.e. the `yeast_codon_table.tsv`. The script calculates the normalised number of reads within a window surrounding each occurrence of the feature of interest (e.g. a codon pair) which are then overlayed to generate a single plot. Normalisation is carried out within a set window of codons adjacent to the feature of interest. The default value of this window is currently 5 codons either side of the feature of interest. 
```

Comment:
`Read counts are fetched from the H5 file of a sample and assigns the counts to the cooresponding codon pair position` -> `Read counts are fetched from the H5 file of a sample by [FUNCTION] and then [FUNCTION] assigns the counts to the corresponding codon pair position`